src/languages/lib/swift.js

```diff
@@ -0,0 +1,285 @@
+import { concat, either } from '../../lib/regex';
+
+// Keywords that require a leading dot.
+export const dotKeywords = [
+  /Protocol\b/, // contextual
+  /Type\b/, // contextual
+];
+
+// Keywords that may have a leading dot.
+export const optionalDotKeywords = [
+  /init\b/,
+  /self\b/
+];
+
+// Regular keywords and literals.
+export const keywords = [
+  /\bAny\b/,
```

Comment:
This \b \B repetition would be nice to abstract and avoid repeating on every line.

Reply:
Done.